### PR TITLE
Add Andi's repositories

### PR DESCRIPTION
**Pull request by ashyti**
Hi, in this pull request I want to add my repositories to be tested. Thanks and best regards!

### CODE REVIEW

File: repo/linux/andi-i2c [hunk 1]
```diff
@@ -0,0 +1,9 @@
+url: t://git.kernel.org/pub/scm/linux/kernel/git/andi.shyti/linux.git
```
Comment (rli9) [inline repo/linux/andi-i2c:1]: thanks, do you refer to https or git protocol here?

Similarly is it ok to name this repo as andi-shyti?

File: repo/linux/andi-github [hunk 2]
```diff
@@ -0,0 +1,6 @@
+url: https://github.com/ashyti/linux
```
Comment (rli9) [inline repo/linux/andi-github:1]: thanks, is it ok to name this repo as ashyti? We have one rule [1] for repo name, though it is not that obvious.

[1] https://github.com/intel/lkp-tests/wiki/Repo-Naming-Convention

File: repo/linux/andi-i2c (hunk 1 above)
Comment (ashyti) [reply repo/linux/andi-i2c:1]: Hi, sorry, I have made a copy paste mistake this is git://git.kernel.org/pub/scm/linux/kernel/git/andi.shyti/linux.git

Shall I send a new pull request?

It's OK andi-shyti

File: repo/linux/andi-github (hunk 2 above)
Comment (ashyti) [reply repo/linux/andi-github:1]: I'm OK with ashyti... thanks!

Do you want me to send a new pull request?

Thanks a lot!
Andi

**Comment by ashyti** [comment]
OK, I see that you have added my andi-shyti file, I will work on that, I'm closing this. Thank you!

**Pull request closed without merging**